src/inlineeditorui.js

```diff
@@ -150,6 +158,16 @@ export default class InlineEditorUI extends EditorUI {
 			originKeystrokeHandler: editor.keystrokes,
 			toolbar
 		} );
+
+		// Set toolbar's max-width on the initialization and update it on the editable resize,
+		// if 'shouldToolbarGroupWhenFull' in config is set to 'true'.
+		if ( !this._toolbarConfig.shouldNotGroupWhenFull ) {
+			const widthObserver = getResizeObserver( ( [ entry ] ) => {
+				this._setToolbarMaxWidth( entry.contentRect.width );
```

Comment:
A funny fact: I noticed that the `contentRect` from the native observer is a `DOMRect` without paddings, so it does not correspond to [`getBoundingClientRect`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect).

![image](https://user-images.githubusercontent.com/1099479/73079849-1b8e2000-3ec5-11ea-8e10-c5a34f4d37f0.png)

![image](https://user-images.githubusercontent.com/1099479/73079871-26e14b80-3ec5-11ea-91d7-9db678a1ce0a.png)

and, as a result, the `maxWidth` set on the toolbar is smaller than it could be. Check out `console.log( entry.contentRect, editableElement.getBoundingClientRect() );`, these are different things

![image](https://user-images.githubusercontent.com/1099479/73080166-aff88280-3ec5-11ea-9b1a-ad5deec9260b.png)

To include padding we'd need the `borderBoxSize`, but it's [available in Firefox only ATM](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry) (this is what it looks like in FF).

![image](https://user-images.githubusercontent.com/1099479/73080571-55abf180-3ec6-11ea-9aac-3e8f6b33d4c6.png)

For now, let's use all the space we have by using our `Rect` implementation that is safe and includes paddings:

```js
const widthObserver = getResizeObserver( () => {
	const editableRect = new Rect( editableElement );

	view.toolbar.maxWidth = toPx( editableRect.width );
} );
```